tests/unit/test_workflows.py

```diff
@@ -665,10 +667,12 @@ def test_from_dict_with_custom_payload(self):
             assert wf.processing_limit_ts == 5000
             assert wf.custom_payload == {'foo': 'bar'}
 
+            # order of calls may not be preserved in older versions of Python
+            # as dict order was non-deterministic < 3.7
```

Comment:
< 3.6 in fact

Reply:
Almost https://stackoverflow.com/questions/39980323/are-dictionaries-ordered-in-python-3-6/39980744#39980744

It's considered as feature since 3.7

Reply:
Ok true :) I just haven't use different implementation than CPython so I didn't noticed it and know that it was only implementation not language feature.